src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheck.java

```diff
@@ -66,6 +78,14 @@
  * r = 5; int t; //violation here
  * </pre>
  * <p>
+ * An example of check with enabled processing resources
```

Comment:
`resources` => `resources:`

I feel that this wording isn't enough for the users to fully understand.
Something like: `An example of how to configure the check to treat resources in a try statement as statements to require them on their own line.`
Property description should be updated in a similar fashion.

@pbludov Please confirm.

Reply:
added colon.

About description - may be it is  better to use something like:
```
An example of configuration to check resource declarations in a try statement. Every resource declaration should be on new line
```
?

Reply:
Most of our wordings try to start with `An example of how to configure the check to ...` so I think we should stick with that.

Reply:
Please also expand on same wording used in other areas.

Reply:
> Please also expand on same wording used in other areas.

what areas do you mean?

Reply:
Property description. `Enable resources processing` is not a very descriptive description. `treatTryResourcesAsStatement` tells me more then the description.

Reply:
corrected